scripts/koji/pyproject.toml

```diff
@@ -0,0 +1,47 @@
+[project]
```

Comment:
I' don't much point of this, if it's not going to install any script, In any case I'd rather only have a single pyproject.toml file at the root, it can be limited to only detect a single package from the koji directory:

```
[project]
name = "xcp"
requires-python = ">= 3.11, <4"
dynamic = ["version"]
readme = "README.md"
dependencies = ["specfile >= 0.34.0"]

[project.scripts]
"koji-xcp" = "koji.koji_build:main"

[build-system]
requires = ["setuptools >= 77.0.3", "setuptools-scm>=8"]
build-backend = "setuptools.build_meta"

[tool.setuptools]
package-dir = {"" = "scripts"}
packages = ["koji"]

[tool.setuptools_scm]

```

Reply:
In my opinion we should split this project in their own repository.
That would be easier to properly tag versions, make releases, run the CI,…

Reply:
Absolutely

Reply:
If we go that way, maybe there is no point in keeping the `koji_build.py` compat script?